modules/text2vec-gpt4all/vectorizer/todo_move_out_of_here_movements.go

```diff
@@ -0,0 +1,68 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package vectorizer
+
+import (
+	"fmt"
+
+	libvectorizer "github.com/weaviate/weaviate/usecases/vectorizer"
+)
+
+// CombineVectors combines all of the vector into sum of their parts
+func (v *Vectorizer) CombineVectors(vectors [][]float32) []float32 {
+	return libvectorizer.CombineVectors(vectors)
+}
+
+// MoveTo moves one vector toward another
+func (v *Vectorizer) MoveTo(source []float32, target []float32, weight float32,
+) ([]float32, error) {
+	multiplier := float32(0.5)
```

Comment:
Where do we get the value `0.5`?

Reply:
This is the original implementation of `nearText` that is used by all `text2vec` modules it is being used to compute the result vector using this formula:
```
out[i] = sourceItem + weight*multiplier*(sourceItem-target[i])
```

Reply:
Ok, interesting. Thanks